src/main/java/net/elytrium/limboauth/handler/AuthSessionHandler.java

```
@@ -442,7 +442,7 @@ private void finishAuth() {
     } catch (SQLException e) {
       throw new SQLRuntimeException(e);
     } catch (Throwable e) {
-      e.printStackTrace();
+      // Error during auth finalization is silently caught to avoid disconnecting already authenticated player
```

It’s bad practice to ignore the exception